doc/api/assert.md

```diff
@@ -284,6 +289,12 @@ assert.fail('boom');
 
 assert.fail('a', 'b');
 // AssertionError: 'a' != 'b'
+
+function suppressFrame() {
+  assert.fail('a', 'b', 'whoops', '!==', suppressFrame);
+}
+suppressFrame();
+// AssertionError: 'a' !== 'b'
```

Comment:
since you gave a message it goes:
```js
AssertionError [ERR_ASSERTION]: whoops
    at repl:1:1
    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
```